styles.css

```diff
@@ -357,7 +357,7 @@ body {
     color: var(--nav-color);
     border: none;
     border-radius: 5px;
```

Comment:
For border radius, see https://stackoverflow.com/questions/23991054/does-rem-make-sense-on-border-radius. parent is "rem" unit, then the border radius should be in rem only.

Reply:
yeah Thanks